packages/fx-core/package.json

```diff
@@ -79,6 +79,7 @@
     "md5": "^2.3.0",
     "mime": "^2.5.2",
     "mustache": "^4.2.0",
+    "neverthrow": "^3.2.0",
```

Comment:
`import { Result } from "@microsoft/teamsfx-api"` so no need to change package.json.

Reply:
Cool, will add it

Reply:
Added